lib/middlewares/src/metering.rs

```diff
@@ -183,6 +240,16 @@ pub fn get_remaining_points(instance: &Instance) -> u64 {
         .expect("`remaining_points` from Instance has wrong type")
 }
 
+pub fn get_error(instance: &Instance) -> MeteringError {
+    instance
+        .exports
+        .get_global("error_code")
+        .expect("Can't get `error_code` from Instance")
+        .get()
+        .try_into()
+        .expect("`error_code` from Instance has wrong type")
```

Comment:
Something like "(metering) `error_code` has an invalid value" is a bit more clear